### DISCUSSION TITLE
Verbosity

### DISCUSSION THREAD

**Discussion by Salvodif** [germanztz/gnome-shell-extension-rclone-manager#37] [Q&A]
HI
Is there a way to figure out what the extension is doing? because I don't understand if it's working or not

**Comment by germanztz** [comment]
Hi! please, follow the instructions here: [issue-report](https://github.com/germanztz/gnome-shell-extension-rclone-manager?tab=readme-ov-file#issue-report) , and open another detailed issue

Thanks!
